.github/workflows/sdk-update-test.yml

```
@@ -129,13 +140,13 @@ jobs:
     if: failure()
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Generate issue
         run: |
           echo "[At least on of the test workflows](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}) using the latest released SDK ${{ github.event.client_payload.latestTag }} on docs has failed." >> .github/ISSUE_TEMPLATE/sdk-incompatible-with-docs.md
       - name: Create issue
         id: issue
-        uses: peter-evans/create-issue-from-file@v3
+        uses: peter-evans/create-issue-from-file@v4
```

```suggestion
        uses: imjohnbo/issue-bot@v3
```
Automatically adding the ticket on the weekly board will give it more visiblity for planning. There is [sample workflow already on kilt-node](https://github.com/KILTprotocol/kilt-node/blob/develop/.github/workflows/polkadot-version-upgrade-ticket.yml)